http/swagger.yml

```diff
@@ -6195,6 +6285,28 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Source"
+    ScraperTarget:
+      type: object
+      properties:
+        id:
+          type: string
+        links:
+              type: object
+              properties:
+                self:
+                  type: string
+        name:
+          type: string
+        type:
+          type: string
+          enum: [prometheus]
+        url:
+          type: string
+          example:  http://localhost:9090/metrics
+        orgID:
+          type: string
```

Comment:
This Response object should also have org and bucket names.

Reply:
As an aside, if an org or bucket no longer exists (deleted), do you check if they are gone?
If not, please add an issue.